kubernetes/samples/scripts/create-rcu-credentials/README.md

```diff
@@ -0,0 +1,60 @@
+# Creating RCU credentials for a Fusion Middleware domain
+
+This sample demonstrates how to create a Kubernetes secret containing the
+RCU credentials for a Fusion Middleware domain.  The operator expects this secret to be
```

Comment:
The operator does not mandate the credential to be in the form stated as there is the `rcuCredentialsSecret` option which can use any name. I think we changed the wording to with Hugo site to say follows the pattern by default.

Reply:
i am struggling to reword it - do you want to suggest something

Reply:
Maybe we can say "The default secret name will be  `domainUID-rcu-credentials`"